package.json

```diff
@@ -14,6 +14,8 @@
     "ejs": "~2.5.5",
     "finalhandler": "^1.0.3",
     "lodash": "^3.10.1",
+    "optional": "^0.1.3",
```

Comment:
where do we need this dependency?

Reply:
I have no idea, it came with the original modification. Happy to remove it if it's not used.